src/UserInterface/Http/Middlewares/DropInvalidLivewireRequests.php

```diff
@@ -31,6 +35,25 @@ public function handle(Request $request, Closure $next)
         return $next($request);
     }
 
+    /**
+     * The file upload request on Livewire doesn't contain `fingerprint` or
+     * `serverMemo` but a signed URL instead that we can validate using the
+     * built-in `hasValidSignature` method.
+     * (`/livewire/upload-file?expires={timestamp}&signature={signature}`).
+     *
+     * @param \Illuminate\Http\Request $request
+     *
+     * @return bool
+     */
+    private function usesAValidSignature(Request $request) : bool
```

Comment:
```suggestion
    private function hasValidSignature(Request $request) : bool
```

Just name it `hasValidSignature` as it is simply an alias to that with some decoration.